code/aggregate.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import logging
+import os
+import json
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter("[%(asctime)s \t %(message)s]",
+                              "%Y-%m-%d %H:%M:%S")
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Run CBR")
```

Comment:
change description from generic Run CBR to Aggregate Scores from all workers etc